src/main/java/org/opensearch/security/dlic/rest/api/SecuritySSLCertsApiAction.java

```diff
@@ -85,7 +95,10 @@ public String getName() {
 
     @Override
     protected void consumeParameters(RestRequest request) {
+        request.param("nodeId");
+        request.param("cert_type");
         request.param("certType");
+        request.param("in_memory");
```

Comment:
What does this new param do?

Reply:
I reserved it for future. I have an idea to read certs from disks. The use case for such feature is this: 
- Users update certs on all nodes
- Users reloads certs
Before reloading certs they can check that all nodes have new certificates.